src/chainstate/stacks/miner.rs

```diff
@@ -378,7 +521,7 @@ impl StacksBlockBuilder {
     }
```

Comment:
There might be some cleaning left in this file - I think that initially the function `try_mine_tx` was supposed to handle microblocks.

Reply:
Yes -- I think there's a lot in this file that is unused in the stacks node, but removing that and then updating the unit tests appropriately will be a pretty sweeping change. For something like that, which should be more refactoring than anything else, I'd prefer to implement in its own PR -- that way there's not a ton of changes that both change functionality in one place but elsewhere are just refactors.